wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
+Note that semicolon and newline characters are not allowed in `name` and `city` values.
 
-`journalctl -f -u jamulus-headless`
+##### `-L or --licence` 
+Show an agreement window before users can connect
 
-#### Controlling Recording
+##### `-w or --welcomemessage` 
+Welcome message on connect. Can be given as a string or filename, and can contain HTML.
 
-When using the recording function with the `-R` command line option, if the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will recording enabled on/off.
+##### `--serverpublicip` 
+The public IP address of the Server if connecting to a Directory behind the same NAT. See [the Directories guide](Directories) for further information.
 
-To send these signals using systemd, create the following two `.service` files in `/etc/systemd/system`, calling them something appropriate (e.g. `jamulusTogglerec.service`).
+##### `--directoryfile` 
+_Directories only:_ Remember registered Servers even if the Directory is restarted. See [the Directories guide](Directories) for further information.
 
-**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (be aware that doing so is however a potential security risk).
+##### `-f or --listfilter` 
+_Directories only:_ Whitelist Servers registering on the Server. See [the Directories guide](Directories) for further information.
 
-For turning recording on or off (depending on the current state):
-
-~~~
- [Unit]
- Description=Toggle recording state of Jamulus Server
- Requisite=Jamulus-Server
+---
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
-~~~
+### General Server options
+These options can be used regardless of which mode your server is running in (although some may not be relevant for Directories).
 
-For starting a new recording:
+##### `-d or --discononquit` 
+Normally, when a Server is stopped or restarted, any Clients that have not used their “Disconnect” buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.
 
-~~~
- [Unit]
- Description=Start a new recording on Jamulus Server
- Requisite=Jamulus-Server
+##### `-F or --fastupdate` 
+Reduces latency if Clients connect with “Enable Small Network Buffers” option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
-~~~
+##### `-l or --log` 
+Enable logging, set path and file name
 
-_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+##### `-m or --htmlstatus` 
```

Comment:
I think this should be tagged as deprecated.  Last time it was reviewed, I was the only person who said it should stay - with a view to moving off it - and now I've moved.

Reply:
Looks like @mcfnord just did.